targets/targets.json

```diff
@@ -3832,7 +3832,7 @@
             "MX25LM51245G"
         ],
         "components_add": [
-            "OSPIF"
+            "SPINAND"
```

Comment:
I don't think default DISCO_L4R9I is supporting this.

Reply:
Yes, default DISCO_L4R9I is not supporting this. I will remove it.
But, I modify this because I tested this driver on DISCO_L4R9I. 
So far, there is not any board contained SPI NAND Flash which is supported by mbed os. I replaced the default Flash with SPI NAND Flash MX31LF4GE4BC on DISCO_L4R9I.

Reply:
you could play with your local mbed_app.json file :-)